partiql-tests-data/eval/query/order-by.ion

```diff
@@ -232,6 +232,122 @@ simple::[
       output:[{'supplierId': 10, n: 3}, {'supplierId': 11, 'n':2}]
     }
   },
+  // should order by aggregation function directly
+  {
+    name:"order by aggregation function",
+    statement:"SELECT supplierId, MAX(price) FROM products GROUP BY supplierId ORDER BY MAX(price) DESC",
```

Comment:
Should include `supplierId` in the ORDER BY since the max value is the same between the `supplierId`s.